### ISSUE TITLE
Synchrotron radiation plugin: Operations out of range

### ISSUE THREAD

**Issue by steindev** [ComputationalRadiationPhysics/picongpu#5036]
Hey, when compiling a simple setup using the standard `speciesXXX.param` files I receive the following warnigs:
```bash
include/picongpu/../picongpu/particles/synchrotron/AlgorithmSynchrotron.hpp(178): warning #222-D: floating-point operation result is out of range
                          * (ELECTRON_CHARGE * ELECTRON_CHARGE * ELECTRON_MASS * SPEED_OF_LIGHT
                                                               ^

Remark: The warnings can be suppressed with "-diag-suppress <warning-number>"

include/picongpu/../picongpu/particles/synchrotron/AlgorithmSynchrotron.hpp(179): warning #222-D: floating-point operation result is out of range
                             / (HBAR * HBAR * EPS0 * 4._X * PI));
```

cc @FilipO28555

**Comment by finnolec** [comment]
I'm running into the same warnings when I compile a simulation with millimeter-cell sizes

**Comment by steindev** [comment]
> I'm running into the same warnings when I compile a simulation with millimeter-cell sizes

This is also the situation I have.

**Comment by BrianMarre** [comment]
@finnolec and @steindev Just because I am paranoid what value does `TYPICAL_PARTICLES_PER_MACROPARTICLE` have in your setups?

> is set in unit.param by default depending on `TYPICAL_PPC` and `BASE_DENSITY` from `density.param` and `particle.param` respectively

**Comment by finnolec** [comment]
We're using the default `BASE_DENSITY_SI = 1.e25` and `TYPICAL_PARTICLES_PER_CELL = 2u` but we both have vacuum simulation boxes. The solution would probably be to manually adapt the `unit.param` to use sane defaults... if you would ever use the synchrotron plugin with millimeter cell sizes in vacuum

**Comment by finnolec** [comment]
Maybe the plugin does not need to be compiled without synchrotron particles

**Comment by psychocoderHPC** [comment]
The error still exists, can someone please have a look into it.

**Comment by psychocoderHPC** [comment]
should be fixed by #5069
